src/dataset/dataset_coco.py

```diff
@@ -312,3 +316,41 @@ def extract_patches(self, output_dir: str, patch_size: int, stride: int, min_are
                 pass
 
         patch_annotations.save(output_path=os.path.join(output_dir, "annotations", "annotations.json"))
+
+
+def extended_dimensions(patch_size: int, stride: int, image_shape: Tuple[int, ...]) -> Tuple[int, ...]:
```

Comment:
Missing docstring.
Nitpick: [Codium](https://www.codium.ai/?gclid=Cj0KCQiAwbitBhDIARIsABfFYIKlKxIsXaoVdnfmz74RudHPs7huga5B3itNq1u_stXfBHDLGkYSxWQaArevEALw_wcB) extension can do it automatically for you using a GPT model.

Reply:
Not a nitpick. I was planning to do it. Thanks for the suggestion of the extensions, it will accelerate things, for sure.